### ISSUE TITLE
Multine table gets too many lines

### ISSUE THREAD

**Issue by mirh** [jgm/pandoc#7404]
So, yet another latex tables regression from 2.11.3 (I think #6842 maybe?)

```
----------------------------------------------------------------------------------------------------
 Attributes→   A1   A2   A3   A4   A5   A6   A7   A8   A9   A10   A11   A12   A13   A14   A15   A16   
 Items  ↓            
------------- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----- ----- ----- ----- ----- ----- ----- 
```
This header used to result in just two lines of text (with the A# columns row being aligned on the bottom). 

Now I get three, since the downward arrow gets a line completely on its own despite the fact that clearly there should be more than enough space in the "first column cell".

**Comment by jgm** [comment]
This gives you a multiline table with just one cell (and the arrow may be wrapped to a line below if everything doesn't fit on one line).  I doubt that's what you want.
If you add a table body
```
----------------------------------------------------------------------------------------------------
 Attributes→   A1   A2   A3   A4   A5   A6   A7   A8   A9   A10   A11   A12   A13   A14   A15   A16   
 Items  ↓            
------------- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----- ----- ----- ----- ----- ----- -----
hi
----------------------------------------------------------------------------------------------------
```
then the header looks like what you are expecting.

It seems you are expecting the bottom row of dashes to be interpreted as the bottom of a header row, and for the table to contain no body.  Instead the bottom row in your example is interpreted as the end of the table, and the table doesn't have a header.  Looking at the documentation, I think this is underspecified. The docs don't really say anything that would lead one to believe it would work in the way you think it should.  Perhaps there has been a change in pandoc's behavior, but since we never intended to allow "header only" multiline tables, I'm not sure it counts as a bug.

**Comment by mirh** [comment]
Oh, lol, the table absolutely has a body. 
I just provided the header because that's the only different thing that got on my nose. 

Also, I'm pretty fine with having two lines (in fact it's what I want). 
The problem is that for some reason, now I have three. 

```
------------------------------------------------------------------------------------------------
 Attributes→      
 Items  
   ↓        A1   A2   A3   A4   A5   A6   A7   A8   A9   A10   A11   A12   A13   A14   A15   A16
------------------------------------------------------------------------------------------------
```
rather than
```
------------------------------------------------------------------------------------------------
 Attributes→      
 Items  ↓   A1   A2   A3   A4   A5   A6   A7   A8   A9   A10   A11   A12   A13   A14   A15   A16
------------------------------------------------------------------------------------------------
```
(this isn't supposed to be markdown code, but ASCII art for the records)

**Comment by jgm** [comment]
That has to do with the relative widths of the columns. Multiline tables encode relative column widths; this is based on the widths of the column header lines in markdown.

If you make the first column a bit wider, it won't wrap.
Try e.g.
```
----------------------------------------------------------------------------------------------------
 Attributes→   A1   A2   A3   A4   A5   A6   A7   A8   A9   A10   A11   A12   A13   A14   A15   A16   
 Items  ↓            
------------- ---- ---- ---- ---- ---- ---- ---- ---- ---- ----- ----- ----- ----- ----- ----- -----
hi
----------------------------------------------------------------------------------------------------
```

**Comment by mirh** [comment]
I see. I actually only needed just an extra dash/space per row for the right results. 

But even though I get that it's impossible for the "text editor appearance" to match the "rendered pdf one", it seems just so odd that with almost twice as much space in the source you'd still need more clearance. 

...on the other hand, I think I also just figured out what actually happened here. 
In 2.11.1 you switched to let {calc} do the table width calculations, which had all my tables slightly resized
This one totally broke instead, and it extended for a good deal outside the page borders. 
In 2.11.3 this was fixed, and together with the previous change, this made for an overall slightly lower table footprint - I guess enough for the first column to become slightly narrower. 
All in all, probably nothing to see here.. So I'm sorry for the bother, but I'm kinda happy that at last I have finally ended the nitpicks/regressions to report.
